src/logic/operate.js

```diff
@@ -0,0 +1,15 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const one = Big(numberOne || 0);
+  const two = Big(numberTwo || 0);
+  let result;
+  if (operation === '+') result = one.plus(two);
+  if (operation === '-') result = one.minus(two);
+  if (operation === 'x') result = one.times(two);
+  if (operation === '÷') result = one.div(two);
+  if (operation === '%') result = one.times(two.div(100));
+  return result.toString();
+};
```

Comment:
We need to handle division by `0`. 
This is no more optional as tagged in https://github.com/ferreirati/mv-react-01-calculator/pull/4#discussion_r364623196 :relaxed: 

See illustration below

![cal-error3](https://user-images.githubusercontent.com/36057474/72137141-8ea67b00-338a-11ea-8383-22c33ecffbd8.gif)

Reply:
Fixed.